resources/warnings.json

```diff
@@ -10557,8 +10557,8 @@
     "url": "https://www.jenkins.io/security/advisory/2022-01-12/#SECURITY-1025",
     "versions": [
       {
-        "lastVersion": "1.19",
-        "pattern": ".*"
+        "lastVersion": "1.398.v550a_8e8ca_e5f",
```

Comment:
This should be last affected version, which seems to be 1.19 according to https://repo.jenkins-ci.org/artifactory/releases/org/jenkins-ci/plugins/batch-task/

Reply:
oh well. Makes sense. I thought it was the first unaffected version :). Fixing